### DISCUSSION TITLE
Logging of the eval results

### DISCUSSION THREAD

**Discussion by vokirill** [Toni-SM/skrl#137] [Q&A]
Hi there! How to configure logging of the evaluation results?

**Comment by Toni-SM** [comment]
Hi @vokirill 

You can just set the `write_interval` configuration to a number higher than 0 to log training/evaluation to Tensorboard: https://skrl.readthedocs.io/en/latest/intro/data.html#configuration

Also, visit the [Examples](https://skrl.readthedocs.io/en/latest/intro/examples.html) to see it in practice

**Reply by vokirill** [reply]
Hi there. This allows me to log training results, but nothing happens during evaluation.
